modules/nodes-cluster-worker-latency-profiles-using.adoc

```diff
@@ -138,3 +139,80 @@ $ oc get KubeControllerManager -o yaml | grep -i workerlatency -A 5 -B 5
 
 To change the medium profile to default or change the default to medium, edit the `node.config` object and set the `spec.workerLatencyProfile` parameter to the appropriate value.
 
+[id="nodes-cluster-worker-latency-profiles-using-at-creation_{context}"]
+== Using worker latency profiles at cluster creation
+
+To implement a worker latency profile in the manifest, follow these steps:
+
```

Comment:
Need a `.Procedure` to start the procedure. 
These steps need to be ordered, so use periods instead of asterisks.

Reply:
You can have only one `.Procedure` section per module.

Reply:
Thank you both, this is done!